Mathlib/Order/LocallyFinite.lean

```diff
@@ -1391,3 +1392,107 @@ theorem Set.finite_iff_bddBelow_bddAbove [Nonempty α] [Lattice α] [LocallyFini
     fun ⟨⟨a,ha⟩,⟨b,hb⟩⟩ ↦ (Set.finite_Icc a b).subset (fun x hx ↦ ⟨ha hx,hb hx⟩ )⟩
 
 end Finite
+
+-- As the instances below are noncomputable,
+-- we make them low priority so when alternative constructions are available they are preferred.
+-- Someone may like to upgrade these instances to computable ones.
+
+noncomputable instance (priority := low) [Preorder α] [LocallyFiniteOrder α] :
+    LocallyFiniteOrder { x : α // x ≤ y } :=
+  LocallyFiniteOrder.ofFiniteIcc fun a b =>
+    Set.Finite.of_finite_image
+      (Set.Finite.subset (Set.finite_Icc ↑a ↑b) (Set.image_subtype_val_Icc_subset a b))
+      Set.injOn_subtype_val
```

Comment:
Can't `Subtype.instLocallyFiniteOrder` already synthesize these?

Reply:
Ugh, it does, assuming you have `open Classical`. I wish we didn't care about computability!

I've removed these.